client/src/pages/index.tsx

```diff
@@ -10,9 +10,11 @@ import LoginPage from "components/login";
 import HomePage from "components/home";
 import { useWithLogin } from "store/slices/login/useWithLogin";
 import { LoginStatus } from "store/slices/login";
+import { loadSentry } from "helpers";
 
 function IndexPage(): JSX.Element {
   const { state: loginState } = useWithLogin();
+  loadSentry();
```

Comment:
i'm not sure this is the place to load sentry, perhaps Kayla or Dilan can provide insight.

Ideally i'd like to have sentry as early as possible, ideally as soon as the html is parsed and first script tag evaluated. But that's not as convenient to configure as this react integration. So can you make sure that sentry init is called before any other react logic kicks in.

Reply:
I wonder if it's just better to go with the gatsby sentry config. With that, sentry is added as a plugin and is loaded very early in the [bootstrapping phase](https://www.gatsbyjs.com/docs/conceptual/overview-of-the-gatsby-build-process/#:~:text=gatsby%20develop%20will%20run%20at,up%20to%20date%20results%20already.).

Reply:
could be, if you have time you can try both. does gatsby use react under the hood?

Reply:
yes gatsby does